apps/mull-ui/src/generated/graphql.tsx

```diff
@@ -0,0 +1,562 @@
+ *
+ * @example
+ * const { data, loading, error } = useDiscoverEventsQuery({
+ *   variables: {
+ *      discoverEventsUserId: // value for 'discoverEventsUserId'
+ *   },
+ * });
+ */
+export function useDiscoverEventsQuery(baseOptions: Apollo.QueryHookOptions<DiscoverEventsQuery, DiscoverEventsQueryVariables>) {
+        return Apollo.useQuery<DiscoverEventsQuery, DiscoverEventsQueryVariables>(DiscoverEventsDocument, baseOptions);
+      }
+export function useDiscoverEventsLazyQuery(baseOptions?: Apollo.LazyQueryHookOptions<DiscoverEventsQuery, DiscoverEventsQueryVariables>) {
+          return Apollo.useLazyQuery<DiscoverEventsQuery, DiscoverEventsQueryVariables>(DiscoverEventsDocument, baseOptions);
+        }
+export type DiscoverEventsQueryHookResult = ReturnType<typeof useDiscoverEventsQuery>;
+export type DiscoverEventsLazyQueryHookResult = ReturnType<typeof useDiscoverEventsLazyQuery>;
+export type DiscoverEventsQueryResult = Apollo.QueryResult<DiscoverEventsQuery, DiscoverEventsQueryVariables>;
+export const GetUsersEventsDocument = gql`
+    query GetUsersEvents($UserId: Int!) {
+  coHostEvents(userId: $UserId) {
+    id
+    title
+    description
+    startDate
+    endDate
+  }
+  hostEvents(userId: $UserId) {
+    id
+    title
+    description
+    startDate
+    endDate
+  }
+}
+    `;
+
+/**
+ * __useGetUsersEventsQuery__
+ *
+ * To run a query within a React component, call `useGetUsersEventsQuery` and pass it any options that fit your needs.
+ * When your component renders, `useGetUsersEventsQuery` returns an object from Apollo Client that contains loading, error, and data properties
+ * you can use to render your UI.
+ *
+ * @param baseOptions options that will be passed into the query, supported options are listed on: https://www.apollographql.com/docs/react/api/react-hooks/#options;
+ *
+ * @example
+ * const { data, loading, error } = useGetUsersEventsQuery({
+ *   variables: {
+ *      UserId: // value for 'UserId'
+ *   },
+ * });
+ */
+export function useGetUsersEventsQuery(baseOptions: Apollo.QueryHookOptions<GetUsersEventsQuery, GetUsersEventsQueryVariables>) {
+        return Apollo.useQuery<GetUsersEventsQuery, GetUsersEventsQueryVariables>(GetUsersEventsDocument, baseOptions);
+      }
+export function useGetUsersEventsLazyQuery(baseOptions?: Apollo.LazyQueryHookOptions<GetUsersEventsQuery, GetUsersEventsQueryVariables>) {
+          return Apollo.useLazyQuery<GetUsersEventsQuery, GetUsersEventsQueryVariables>(GetUsersEventsDocument, baseOptions);
+        }
+export type GetUsersEventsQueryHookResult = ReturnType<typeof useGetUsersEventsQuery>;
+export type GetUsersEventsLazyQueryHookResult = ReturnType<typeof useGetUsersEventsLazyQuery>;
+export type GetUsersEventsQueryResult = Apollo.QueryResult<GetUsersEventsQuery, GetUsersEventsQueryVariables>;
+export const GetParticipatingEventsDocument = gql`
```

Comment:
Can I ask how this was generated?

Reply:
This was generated by running the `npm run gen-graphql` which is a script in the `package.json` file.